docs/contribute/contribute_to_documentation/index.md

```diff
@@ -0,0 +1,69 @@
+# Contributing to ROCm documentation
+
+AMD values and encourages the ROCm community to contribute to our code and
+documentation. This repository is focused on ROCm documentation and this
+contribution guide describes the recommended method for creating and modifying our
+documentation.
+
+While interacting with ROCm Documentation, we encourage you to be polite and
```

Comment:
code of conduct is needed and should be referenced in the future.

Reply:
I'm sorry, I don't understand what you mean?

Reply:
#2409 See a proposed code of conduct. Awaiting legal review for it.

Reply:
So, should this PR wait on the code of conduct? I would suggest no, since this doesn't make anything worse (it just uses the existing text).